CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
+- [doc] Upgrade jinja2 from 3.1.4 to 3.1.5 due to CVE-2024-56201 and CVE-2024-56326
```

Comment:
```suggestion
```

Given it is related with the RTD and not with Cygnus itself, better not mentioning it in CNR.